tests/integration/sheet.js

```diff
@@ -287,12 +275,12 @@ describe('Integration: sheet', () => {
       })
       expect(sheet.toString()).to.be(
         '@media print {\n' +
-        '  .a-3787690649 {\n' +
+        '  .my-special-id {\n' +
         '    float: left;\n' +
         '  }\n' +
         '}\n' +
         '@media screen {\n' +
-        '  .a-3787690649 {\n' +
+        '  .my-special-id {\n' +
```

Comment:
Why, this case should have ensured actually the opposite. But it was late and I made a mistake with a fixed id. Now we don't verify that the id is really same.